README.md

```diff
@@ -13,26 +13,51 @@ itself.
 ![Dashboard UI workloads page](docs/dashboard-ui.png)
 
 ## Deployment
-It is likely that the Dashboard is already installed on your cluster. Check with the following command:
+Provided instructions are compliant with kubernetes 1.6+. If you plan to deploy dashboard on older kubernetes version please
+follow [these instructions](#older-kubernetes-versions).
```

Comment:
I like this idea, but I would move to separate page. The landing page should be as easy as possible
You could do similar with multi-tenant setup